sdk/python/lib/pulumi/provider/experimental/ComponentProvider.md

```diff
@@ -0,0 +1,185 @@
+# Component Provider
```

Comment:
Fine for now but we should refine language before we publish this widely. Do we want to use "provider", "package", or drop that all? TBD.